src/OrchardCore.Modules/OrchardCore.Contents/placement.json

```diff
@@ -6,16 +6,6 @@
 */
 
 {
-  "Content_PublishButton": [
-    {
-      "place": "Actions:5" // Same as in the driver
```

Comment:
Are you sure the driver's the one taking effect with `Action:10`?

Reply:
apart from this line : 
https://github.com/OrchardCMS/OrchardCore/blob/0c00316259d47f2c7abf084d3fc7b0d7f5a48ab9/src/OrchardCore.Modules/OrchardCore.Contents/Drivers/ContentsDriver.cs#L104

where else would it be used  ?

Reply:
I mean, are you sure that when the layout is rendered with the current `main` code, then the placement that will take effect is the one defined in the driver, with `Action:10`, and not this one with `Action:5`?

Reply:
I tested in the admin **edit content page** and **widget edit page** and it works the same when applying this pr.

![image](https://github.com/OrchardCMS/OrchardCore/assets/1516981/de043ddc-f4b6-4f06-ad3a-23d214c1b18e)


![image](https://github.com/OrchardCMS/OrchardCore/assets/1516981/270b8faf-e9e4-4068-9265-47403a13f8b3)

Reply:
OK, thank you!